### PR TITLE
(aptana-studio) Migrated to AU [AU aptana-studio]

### LINKED ISSUE #179: aptana-studio is failing verifier
https://chocolatey.org/packages/aptana-studio/3.6.1

### LINKED ISSUE #242: Can't install aptana-studio
I can't install aptana-studio by using chocolatey.
It seems to fail at download from

``` xml
<TextualContent>http://sourceforge.net/projects/aptana.mirror/files/Aptana Studio{version}/AptanaStudio{majorVersion}Setup{version}.exe/download</TextualContent>
```

I think it must be

``` xml
<TextualContent>https://github.com/aptana/studio{majorVersion}/releases/download/v{version}/AptanaStudio{majorVersion}Setup{version}.exe</TextualContent>
```

### PR DESCRIPTION
**Pull request by AdmiringWorm**
* Modernized install script + fixed silent args
* Modernized uninstall script + Fixed logic
* Added AU update script
* Updated metadata
  * Added project source url
  * Updated description
  * Added missing javaruntime dependency
  * Added missing git dependency
  * Updated release notes url
  * Added bug tracker url
  * Updated icon url
  * Added new tags

fixes #242
fixes #179

### CODE REVIEW

**Comment by majkinetor** [comment]
It does seem like the project is abandoned to me.

**Comment by AdmiringWorm** [comment]
Forgot to mention, this can't be pushed to chocolatey until the core extension is pushed and accepted.

File: automatic/aptana-studio/aptana-studio.nuspec [hunk 1]
```diff
@@ -1,27 +1,65 @@
...
+  Easily put your projects under git source code control.
+  Collaborate with team members thru merge, pull and push actions to remote repositories such those hosted on Github.
+  Facilitates git-based deployments.
+* __Built-in Terminal__
+  Quickly access a command line terminal for execution of operating system commands and language utilities such as gem, rake, etc.
+* __IDE Customization__
+  Setup your development environment exactly the way you want it by extending the core capabilities through scripting of custom commands.
+  Studio ships with hundreds of commands but always presents them in context based on the type of file your are editing.
+
+## Note
```
Comment (majkinetor) [inline automatic/aptana-studio/aptana-studio.nuspec]: Note**s** 

(not terribly important, for next time)

**Comment by majkinetor** [comment]
I closed those 2 issues, lets wait for @gep13 to public core extension, I am not sure if he wanted to do something else with it.

**Comment by gep13** [comment]
>@majkinetor said...
I closed those 2 issues, lets wait for 

That wasn't necessary.  Given that @AdmiringWorm put the following in the initial post:

image

Those issues would have been closed automagically when the PR was merged 😄

File: automatic/aptana-studio/aptana-studio.nuspec (hunk 1 above)
Comment (AdmiringWorm) [inline automatic/aptana-studio/aptana-studio.nuspec]: missed that, yeah. next time

**Comment by AdmiringWorm** [comment]
@gep13 Altough I added them to the initial post, that wouldn't have mattered if I hadn't added them to
the commit message.
Issue numbers in Pull requests only reference.

**Comment by gep13** [comment]
>@AdmiringWorm said...
@gep13 Although I added them to the initial post, that wouldn't have mattered if I hadn't added them to
the commit message.
Issue numbers in Pull requests only reference.

Nope, that is not true 😄 

You can automatically close an issue by writing what you did, i.e. `fixes #123` or `resolves #123`

https://github.com/blog/1506-closing-issues-via-pull-requests

**Comment by AdmiringWorm** [comment]
@gep13 Really?
hmm, odd that it didn't work in the [freecad PR](https://github.com/chocolatey/chocolatey-coreteampackages/pull/373) then.

**Comment by gep13** [comment]
Yip, was just away to type the same thing 👍

**Comment by majkinetor** [comment]
> That wasn't necessary. Given that @AdmiringWorm put the following in the initial post:
I thought that it didn't happen for some reason, didn't know in this case it happens on PR merge.

**Comment by gep13** [comment]
Yip, it is a very useful feature.  I fluctuate between liking it and not liking it 😄

**Pull request merged**